### PR TITLE
clarify how to pass options to recast

### PR DESCRIPTION
**Pull request by jamischarles**
See #55 

Will happily modify if you have improvement suggestions on the format.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -281,6 +281,16 @@ jscodeshift(ast).findIdentifiers().logNames();
 jscodeshift(ast).logNames(); // error, unless `ast` only consists of Identifier nodes
 ```
 
+### Passing options to [recast]
+You may want to change some of the output settings (like setting `'` instead of `"`). This can be done by passing config options to [recast].
```
Comment (wincent) [inline README.md]: Looks like elsewhere in this file we're hard-wrapping long lines. Care to match that?

File: README.md [hunk 2]
```diff
@@ -281,6 +281,16 @@ jscodeshift(ast).findIdentifiers().logNames();
...
+
+```js
+.toSource({quote: 'single'}); // sets strings to use single quotes in transformed code.
+```
+
+More on config options [here](https://github.com/benjamn/recast/blob/52a7ec3eaaa37e78436841ed8afc948033a86252/lib/options.js#L61)
+
+
```
Comment (wincent) [inline README.md]: Excess blank line here.

File: README.md [hunk 3]
```diff
@@ -281,6 +281,16 @@ jscodeshift(ast).findIdentifiers().logNames();
 jscodeshift(ast).logNames(); // error, unless `ast` only consists of Identifier nodes
 ```
 
+### Passing options to [recast]
```
Comment (wincent) [inline README.md]: Needs a blank line after the heading.

**Comment by wincent** [comment]
Thanks! Looks good to me other than a few nits noted inline.

I'll leave this up here for a bit to give @fkling time to see it and chime in if he wants.

**Comment by jamischarles** [comment]
@wincent Addressed your comments and force pushed. Let me know if there's anything else.

**Comment by fkling** [comment]
lgtm, thanks!

**Pull request merged**